controllers/suite_test.go

```diff
@@ -241,24 +333,38 @@ func deleteClusterVersion() {
 	Expect(k8sClient.Delete(ctx, instance)).Should(Succeed())
 }
 
-func clusterPrep() {
+func createDeployment(ctx context.Context, deployment *appsv1.Deployment) {
+	Expect(k8sClient.Create(ctx, deployment)).Should(Succeed())
+}
+
+func deleteDeployment(ctx context.Context, deployment *appsv1.Deployment) {
+	Expect(k8sClient.Delete(ctx, deployment)).Should(Succeed())
+}
+
+// func createPVC(ctx context.Context, pvc *corev1.PersistentVolumeClaim) {
```

Comment:
should we keep it or remove this comment?

Reply:
Yes, I removed these comments.